tests/posix/threads_base/src/pthread.c

```diff
@@ -460,7 +457,7 @@ ZTEST(pthread, test_pthread_descriptor_leak)
 	}
 }
 
-ZTEST(pthread, test_sched_getparam)
+ZTEST(posix_threads_base, test_sched_getparam)
```

Comment:
This test belongs in the `xsi_realtime` testsuite with the `_POSIX_PRIORITY_SCHEDULING` Option.